ihmc-graphics/build.gradle.kts

```diff
@@ -45,7 +45,8 @@ jmonkeyengineTestDependencies {
 }
 
 libgdxDependencies {
-   api(ihmc.sourceSetProject("main"))
```

Comment:
This had the effect of bringing in sensor processing and the main source set, which brought in this stuff, too.
![image](https://github.com/user-attachments/assets/8813bbda-ca9b-43fa-b372-c0e5b51a70fd)